templates/structure/sponsor.php

```diff
@@ -1,11 +1,16 @@
 <?php declare(strict_types=1) ?>
 <?php /** @var \CascadiaPHP\Site\Template\Template $this */ ?>
 
+<?php if (isset($link)): ?>
+    <a href="<?= $link ?>">
+<?php endif ?>
+
 <div class="flex p2 sponsor">
-    <amp-layout layout="responsive" width="1" height="1" sizes="(min-width: 40em) 200px, calc(100vw - 6em)" class="circle b-<?= $borderColor ?? 'black' ?> bg-<?= $backgroundColor ?? 'white' ?>">
+    <amp-layout layout="responsive" width="1" height="1" sizes="(min-width: 40em) calc((1440px - 14em) / 3), calc(100vw - 6em)" class="shadow-md circle b-<?= $borderColor ?? 'black' ?> bg-<?= $backgroundColor ?? 'white' ?>">
```

Comment:
I'm unfamiliar with this but I assume it's acceptable to mix UOMs (units of measure) in calculations? Seems strange to me.

Reply:
[yep](https://developer.mozilla.org/en-US/docs/Web/CSS/calc)